doc/developing.rst

```diff
@@ -79,7 +79,8 @@ You can run the tests using `py.test`_::
 
 To generate test coverage information as HTML do::
 
-  $ py.test --cov morepath --cov-report html
```

Comment:
This change involves turning one line into two. Why not ``py.test --cov --cov-report html``?